torch/_export/serde/schema.py

```diff
@@ -27,6 +27,10 @@ class ScalarType(IntEnum):
     COMPLEXDOUBLE = 11
     BOOL = 12
     BFLOAT16 = 13
+    FLOAT8_E5M2 = 14
+    FLOAT8_E4M3FN = 15
+    FLOAT8_E5M2FNUZ = 16
+    FLOAT8_E4M3FNUZ = 17
```

Comment:
Can we stick to the numbering [here](https://github.com/pytorch/pytorch/blob/cab90b0049f75a1c6d9ca034e5011fa170a944e8/c10/core/ScalarType.h#L56-L93) (23-26). We should fix the other numberings eventually too..